includes/Abilities/Image/Generate.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Image generation WordPress Ability implementation.
+ *
+ * @package WordPress\AI
+ */
+
+declare( strict_types=1 );
+
+namespace WordPress\AI\Abilities\Image;
+
+use WP_Error;
+use WordPress\AI\Abstracts\Abstract_Ability;
+use WordPress\AI_Client\AI_Client;
+use WordPress\AiClient\Files\Enums\FileTypeEnum;
+
+use function WordPress\AI\get_preferred_image_models;
+
+/**
+ * Image generation WordPress Ability.
+ *
+ * @since 0.1.0
+ */
+class Generate extends Abstract_Ability {
```

Comment:
Nit, but I think more self-explanatory class names are preferable in terms of encouraging more readable code. I realize this is in an `Image` namespace, but when you use the class name on its own somewhere, that context is lost when looking at the concrete line of code.

So I'd suggest we use `Generate_Image`, despite this already being in an `Image` namespace (which makes sense to keep).

Reply:
Makes sense, updated in https://github.com/WordPress/ai/pull/134/changes/92308500bb7ba1e4620a8e478cef794ee8f60294